### PR TITLE
chore: merge `dev` to `main`

### PR DESCRIPTION
**Pull request by Ushie**

### CODE REVIEW

File: pubspec.yaml
Comment (ILoveOpenSourceApplications) [inline pubspec.yaml:1]: Shouldn't it be 
```java
version: 1.5.1+100500001
```  
?

**Comment by ILoveOpenSourceApplications** [comment]
I didn't mean to be a reviewer, just a simple doubt considering the previous versions. Also after this commit was pushed, the previous release has also disappeared.

File: pubspec.yaml
Comment (validcube) [reply pubspec.yaml:1]: It should :skull:

Comment (validcube) [reply pubspec.yaml:1]: > Shouldn't it be 
> ```java
> version: 1.5.1+100500001
> ```  
> ?

@Ushie

**Review by validcube — CHANGES_REQUESTED** [review]
See comment

File: pubspec.yaml
Comment (Ushie) [reply pubspec.yaml:1]: The versioncode doesn't matter, it was the way it was for the Semantic Releases CI, which we no longer use, and as it doesn't matter there's no need to put more effort than just bumping it by 1

**Comment by Ushie** [comment]
> I didn't mean to be a reviewer, just a simple doubt considering the previous versions. Also after this commit was pushed, the previous release has also disappeared.

Both releases contained upstream bugs, the correct way would've been to downgrade ReVanced Patcher to 11.0.4 and make a new release though, but it's fine

**Comment by ILoveOpenSourceApplications** [comment]
Why downgrade the ReVanced Patcher version though?

**Comment by Ushie** [comment]
The ReVanced Patcher is where the issues where introduced in

**Pull request merged**